LICENSE

```diff
@@ -1,5 +1,6 @@
 MIT License
 
+Copyright (c) 2018 TheTrunk
```

Comment:
If we want to change this, shouldn't this Copyright move under the Hush team?

Reply:
Agree